README.md

```diff
@@ -104,6 +104,10 @@ To check the status of your IPsec VPN server, you can pass `ipsec status` to you
 docker exec -it l2tp-ipsec-vpn-server ipsec status
 ```
 
+### Scaleway user information
+
+Scaleway use own modified kernel version `4.4` by default. This kernel isn't compatible with IPsec. If you want to use IPsec VPN server on Scaleway VPS you should switch version of kernel (version `4.8` or `higher`).
```

Comment:
Can you add a link to Scaleway website on the Scaleway string please. So there is no misunderstanding on what is Scaleway.

Reply:
@kirill-zak : Are you a Scaleway employee or someone related to the Scaleway company ?